CHANGELOG.md

```diff
@@ -2,8 +2,91 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 43 end 2016-07-11
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+43+Ending+July+11%2C+2016%22+label%3Aenhancement)
+
+- Automate
+  - Service resolution based on Provision Order
+  - Added /System/Process/MiqEvent instance
+  - Added Provider refresh call to Amazon retire state machine in Pre-Retirement state
+  - Service Dialogs: Added ‘Visible’ flag to all dialog fields
+  - Class Schema allows for EMS, Host, Policy, Provision, Request, Server, Storage, and VM(or Template) datatypes
+    - Value is id of the objects
+    - If object is not found, the attribute is not defined.
+- Platform: Custom attributes
+  - Now selectable as a column
+  - Support in report filters and charts
+- Providers
+  - Amazon: Public Images Filter
+  - Networking
+      - Separate Google Network Manager
+      - NFV: VNFD Templates and VNF Stacks
+  - Pluggability: SupportsFeatureMixin
+      - supports :provisioning
+      - supports : discovery
```

Comment:
I'm not sure sure SupportsFeature stuff should go into the CHANGELOG.  Ultimately, it's just refactoring and means nothing to an out-side in user.